repose-aggregator/components/filters/keystone-v2-filter/src/test/scala/org/openrepose/filters/keystonev2/KeystoneV2FilterPrepTest.scala

```diff
@@ -129,7 +127,7 @@ class KeystoneV2FilterPrepTest extends FunSpec with Matchers with MockitoSugar w
   describe("when the configuration is updated") {
     it("sets the current configuration on the filter asserting the defaults and initialized is true") {
       val filter = new KeystoneV2Filter(mock[ConfigurationService], mock[AkkaServiceClientFactory], mock[AtomFeedService], mockDatastoreService)
-      filter.isInitialized shouldNot be(right = true)
```

Comment:
Good point. Fixed. Along with updating the uses of `should be(*)` and `should equal(*)` to the more widely used (at least in Repose) `shouldBe *` and `shouldEqual *`.